File: 2358. Number of Ways to Split Array/Solution.py
class Solution:
    def waysToSplitArray(self, nums: List[int]) -> int:
        n = len(nums)
        ans = 0
        total_sum = sum(nums)
        left = 0

        for i in range(n-1):
            print(i)
            left += nums[i]
            right = total_sum - left
            if left >= right:
                ans += 1
        return ans

File: 2358. Number of Ways to Split Array/Solution.go
func waysToSplitArray(nums []int) int {
    n := len(nums)
        ans := 0
        total_sum := 0
        left := 0
        for _, i := range nums{
            total_sum += i
        }

        for j:= 0; j<n-1; j++ {
            fmt.Println(j)
            left += nums[j]
            right := total_sum - left

            if left >= right{
                ans += 1
            }
        }
        return ans
}